lib/docraptor/models/doc.rb

```diff
@@ -1,6 +1,19 @@
+=begin
```

Comment:
I have chosen not to remove these bits of whatever crap from the various files where it appears. There is a license remover bit in #7 that could do it pretty easily if desired.